### PR TITLE
Updated node controller for taints and tolerations.

### PR DESCRIPTION
**Pull request by gyliu513**
Fixed part of #2737 , depend on https://github.com/kubernetes/kubernetes.github.io/pull/2774 , will update this PR once https://github.com/kubernetes/kubernetes.github.io/pull/2774 got merged.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2777)

### CODE REVIEW

File: docs/admin/node.md [hunk 1]
```diff
@@ -165,6 +165,11 @@ completely unhealthy (i.e. there are no healthy nodes in the cluster). In such
 case, the node controller assumes that there's some problem with master
 connectivity and stops all evictions until some connectivity is restored.
 
+In Kubernetes 1.6, we introduced the logic for the node controller to handle
```
Comment (cmluciano) [inline docs/admin/node.md]: Kubernetes 1.6 introduces logic...

File: docs/admin/node.md [hunk 2]
```diff
@@ -165,6 +165,11 @@ completely unhealthy (i.e. there are no healthy nodes in the cluster). In such
...
+`taints` and `tolerance`. Basically, `taints` go on nodes while `tolerations`
```
Comment (cmluciano) [inline docs/admin/node.md]: Consider rephrasing this sentence.

e.g. (Taints are applied to nodes and tolerations are applied to pods.)

Comment (aveshagarwal) [inline docs/admin/node.md]: s/while/and

**Comment by aveshagarwal** [comment]
FYI: @mburke5678

File: docs/admin/node.md (hunk 2 above)
Comment (mburke5678) [inline docs/admin/node.md]: +1

Comment (mburke5678) [inline docs/admin/node.md]: Is "basically" acceptable language for upstream? You could probably remove it without affecting the meaning.

Comment (cmluciano) [inline docs/admin/node.md]: I agree. I think basically should be removed.

**Comment by caesarxuchao** [comment]
@gmarek i reassigned to you since you are listed at the owner of pkg/controller/node.

File: docs/admin/node.md (hunk 1 above)
Comment (gyliu513) [inline docs/admin/node.md]: But here https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/admin/node.md#node-controller we did have similar sentence as 

```
In Kubernetes 1.4, we updated the logic of the node controller
```

Shall we keep consistent?

File: docs/admin/node.md (hunk 2 above)
Comment (gyliu513) [inline docs/admin/node.md]: Done

File: docs/admin/node.md (hunk 1 above)
Comment (davidopp) [inline docs/admin/node.md]: Please rewrite this as:

```
Starting in Kubernetes 1.6, the NodeController is also responsible for evicting pods that are running
on nodes with `NoExecute` taints, when the pods do not tolerate the taints. Additionally, as an alpha
feature that is disabled by default, the NodeController is responsible for adding taints
corresponding to node problems like node unreachable or not ready. See
[this documentation](/docs/user-guide/node-selection/index.md) for details about `NoExecute`
taints and the alpha feature.
```

Comment (gyliu513) [inline docs/admin/node.md]: Done, thanks @davidopp

**Comment by davidopp** [comment]
LGTM

**Pull request merged**